packages/runner-ct/src/specs/spec-file.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react'
+import { observer } from 'mobx-react'
+import State from '../lib/state'
+
+interface SpecFileProps {
+  state: State,
+  // eslint-disable-next-line
+  spec: import('./make-spec-hierarchy').SpecFile
+}
+
+export const SpecFile = observer(
+  function SpecFile ({ spec, state }: SpecFileProps) {
+    const isActive = state.runMode === 'multi'
+      ? state.multiSpecs.some((includedSpec) => includedSpec.absolute === spec.absolute)
+      : spec.name === state.spec?.name
+
+    return (
+      <li
+        key={spec.name}
+        className="spec-list-file"
+        onClick={(e) =>
+          e.shiftKey
+            ? state.addSpecToMultiMode(spec)
```

Comment:
Can we add a feature flag to wrap this behavior? I don't think we want to enable multi spec mode at all for users

Reply:
I don't mind if it's within code. Perhaps we can create a const file with flags and their defaults?

Reply:
Yes we can and this will be awesome. But probably in different PR?

Reply:
I think it's fine to do in another PR

Reply:
OK I put an issue in the internal tracker (CT-195)